src/lib/stitching/mergeSchemas.ts

```diff
@@ -45,33 +47,56 @@ export const incrementalMergeSchemas = (localSchema, version: 1 | 2) => {
   const gravitySchema = executableGravitySchema()
   schemas.push(gravitySchema)
 
-  useStitchingEnvironment(
-    gravityStitchingEnvironment(localSchema, gravitySchema, version)
-  )
+  // Gravity stitching
+  version === 1
```

Comment:
These inline ternaries can be a little brittle - can we convert the following to 

```tsx 
if (version === 1) {
  ...
} else {
  ... 
}